### PR TITLE
Get base entity type name

### PR DESCRIPTION
**Pull request by robin-vagner**
**Problem**
`get_base_entity_type_name` did not handle scenes.

**Solution**
We added a case for scenes.

### CODE REVIEW

File: zou/app/services/shots_service.py [hunk 1]
```diff
@@ -1060,13 +1059,17 @@ def get_shot_versions(shot_id):
 
 
 def get_base_entity_type_name(entity_dict):
-    type_name = "Asset"
```
Comment (frankrousseau) [inline zou/app/services/shots_service.py:1063]: Can you let the variable as declared first?

File: zou/app/services/shots_service.py [hunk 2]
```diff
@@ -1060,13 +1059,17 @@ def get_shot_versions(shot_id):
...
     if is_shot(entity_dict):
         type_name = "Shot"
+    elif is_scene(entity_dict):
```
Comment (frankrousseau) [inline zou/app/services/shots_service.py:1064]: Can you put the scene tests at then end of the condition list? It occurs less often than the two others. Kitsu users don't have scenes.

File: zou/app/services/shots_service.py (hunk 1 above)
Comment (robin-vagner) [reply zou/app/services/shots_service.py]: Yes, it's done.

File: zou/app/services/shots_service.py (hunk 2 above)
Comment (robin-vagner) [reply zou/app/services/shots_service.py]: Makes sense, I fixed it.

**Pull request merged**